api/basic/update-pin.js

```diff
@@ -0,0 +1,53 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb'
+import { DynamoDBDocumentClient, UpdateCommand } from '@aws-sdk/lib-dynamodb'
+
+/**
+ * @param {import('aws-lambda').S3Event} event
+ */
+export async function handler (event) {
+  const {
+    TABLE_NAME: table = '',
+    // set for testing
+    DYNAMO_DB_ENDPOINT: dbEndpoint = undefined
```

Comment:
(nothing blocking) I have seen in w3up pattern where there is a wrapper handler for testing. I am not fan of any of these patterns to test code by injecting things in the actual codebase. If you have ideas, it might be nice to discuss them async and get to a nicer pattern